savvy-transactions/schema.graphql

```diff
@@ -0,0 +1,49 @@
+type Deposit @entity(immutable: true) {
+  id: Bytes!
+  depositor: Bytes! # address
+  yieldToken: Bytes! # address
```

Comment:
All tokens should point to a [Token](https://github.com/savvy-finance/savvy-subgraphs/blob/main/savvy-traderjoe-2/schema.graphql#L1) type